### PR TITLE
fix:Tests for service function isNonReimbursableOrg()

### PR DESCRIPTION
**Pull request by arjunaj5**
https://app.clickup.com/t/85zrpcyvg
[workerB](https://workerb.linearb.io/v2/badge/collaboration-page?magicLinkId=suAnRjX)

### CODE REVIEW

**Review by gokulk16 — APPROVED** [review]

**Review by yksolanki9 — APPROVED** [review]

File: src/app/core/services/payment-modes.service.spec.ts [hunk 1]
```diff
@@ -178,4 +184,16 @@ describe('PaymentModesService', () => {
     });
     expect(trackingService.showToastMessage).toHaveBeenCalledOnceWith({ ToastContent: message });
   });
+
+  describe('isNonReimbursableOrg():', () => {
+    it('should return true if only non reimbursable payment modes in payment mode settings', () => {
```
Comment (Dimple16) [inline src/app/core/services/payment-modes.service.spec.ts:189]: `should return true if only non-reimbursable payment modes exist in the payment mode settings`

File: src/app/core/services/payment-modes.service.spec.ts [hunk 2]
```diff
@@ -178,4 +184,16 @@ describe('PaymentModesService', () => {
...
+      expect(paymentModesService.isNonReimbursableOrg(cccOnlyPaymentModeSettingsParam)).toBeTrue();
+      expect(paymentModesService.isNonReimbursableOrg(cccAndPaidByCompanyPaymentModeSettingsParam)).toBeTrue();
+    });
+
+    it('should return false if reimbursable payment modes in payment mode settings', () => {
```
Comment (Dimple16) [inline src/app/core/services/payment-modes.service.spec.ts:194]: should return false if only reimbursable payment modes exist in the payment mode settings

**Review by Dimple16 — CHANGES_REQUESTED** [review]
can you add the screenshot of the payment-modes service code coverage?

File: src/app/core/services/payment-modes.service.spec.ts (hunk 1 above)
Comment (arjunaj5) [reply src/app/core/services/payment-modes.service.spec.ts]: done

File: src/app/core/services/payment-modes.service.spec.ts (hunk 2 above)
Comment (arjunaj5) [reply src/app/core/services/payment-modes.service.spec.ts]: done 🙇

**Review by Dimple16 — APPROVED** [review]

**Pull request merged**